src/components/cc-oauth-consumer-form/cc-oauth-consumer-form.events.js

```diff
@@ -0,0 +1,57 @@
+import { CcEvent } from '../../lib/events.js';
+
+/**
+ * @typedef {import('./cc-oauth-consumer-form.types.js').OauthConsumerWithoutKeyAndSecret} OauthConsumerWithoutKeyAndSecret
+ */
+
+/**
+ * Dispatched when an OAuth consumer creation is requested.
+ * @extends {CcEvent<OauthConsumerWithoutKeyAndSecret>}
+ */
+export class CcOauthConsumerFormCreateEvent extends CcEvent {
```

Comment:
The naming of events should talk about the manipulated entity and not the thing that initiate the event. So I would have used the following names:

- cc-oauth-consumer-create
- cc-oauth-consumer-update
- cc-oauth-consumer-delete
- cc-oauth-consumer-created

This way, if, for example, another component deletes an oauth consumer, it will make more sense to reuse `cc-oauth-consumer-delete` event than `cc-oauth-consumer-form-delete`.